v1_13_R2/src/main/java/net/citizensnpcs/nms/v1_13_R2/entity/HumanController.java

```diff
@@ -83,6 +83,10 @@ public void run() {
                     team.addPlayer(handle.getBukkitEntity());
 
                     handle.getNPC().data().set(NPC.SCOREBOARD_FAKE_TEAM_NAME_METADATA, teamName);
+
+                    for (Player player : Bukkit.getOnlinePlayers()) {
+                        NMS.sendTeamPacket(player, team);
```

Comment:
You may want to exclude Player NPCs here.

Reply:
I don't think `Bukkit.getOnlinePlayers()` ever contains NPCs (they go in the world player list when `playerlist` is on, never the global one). Other existing code doesn't perform any such check, eg `NMSImpl#sendPacketsNearby` doesn't check for NPCs in the list.